x-pack/plugins/cases/public/components/all_cases/selector_modal/all_cases_selector_modal.test.tsx

```diff
@@ -82,13 +82,4 @@ describe('AllCasesSelectorModal', () => {
 
     expect(res.queryByTestId('cases-metrics-stats')).toBeFalsy();
   });
-
-  it('should show the create case button', async () => {
```

Comment:
The 'Create case' button has been removed which means from ML for example that there is no longer a way to add an ML embeddable to a new case. You can only add to an existing case, where before you had the option to create a new case from this modal. This button needs to be added back, or else changes will be needed in solutions which don't provide separate actions for 'Add to new case' / 'Add to existing case' as the alerts table does.

Reply:
I do think it looks weird when we show the button twice though.

<img width="1195" alt="Screenshot 2023-02-02 at 15 37 14" src="https://user-images.githubusercontent.com/1533137/216354172-3721c1fa-d27b-49f4-9682-455d44b1e9ad.png">

Maybe remove when there are `No cases to display`?

Reply:
yeah, instruction without any action is also a valid empty prompt: https://elastic.github.io/eui/#/guidelines/writing/examples#empty-prompts

I am okay to remove `Create case` from empty prompt

Reply:
Update: This Empty case prompt is used in Modal as well as All Cases table. So I decided to keep `Create case` button in Empty prompt as it might be helpful in some scenarios.